tests/fuzz/corpus/s2n_bike_r1_recv_public_key_fuzz_test/ac9231da4082430afe8f4d40127814c613648d8e

```diff
@@ -0,0 +1 @@
+	
```

Comment:
Empty corpus?

Reply:
This should be fine. Fuzz tests are just trying to generate inputs that cover as many code branches as possible, so if there are any `len > 0` checks, the fuzz tests will likely find it and generate an input to reach that check.